#Zagi
#m: 1.56 #kg
#Jx: 0.1147 #kg-m^2
#Jy: 0.0576 #kg-m^2
#Jz: 0.1712 #kg-m^2
#Jxz: 0.0015 #kg-m^2

#Aerosonde
m: 13.5 #kg
Jx: 0.8244 #kg-m^2
Jy: 1.135 #kg-m^2
Jz: 1.759 #kg-m^2
Jxz: 0.1204 #kg-m^2
S: 0.55 #m^3
b: 2.8956 #m
c: 0.18994 #m
S_prop: 0.2027 #m^2
rho: 1.2682 #kg/m^3
k_motor: 80.0
k_T_p: 0.0
k_Omega: 0.0
e: 0.9
C_L_0: 0.28
C_D_0: 0.03
C_m_0: -0.02338
C_L_alpha: 3.45
C_D_alpha: 0.30
C_m_alpha: -0.38
C_L_q: 0.0
C_D_q: 0.0
C_m_q: -3.6
C_L_delta_e: -0.36
C_D_delta_e: 0.0
C_m_delta_e: -0.5
C_prop: 1.0
M: 50
alpha_0: 0.4712
epsilon: 0.1592
C_D_p: 0.0437
C_n_delta_r: -0.032
C_Y_0: 0.0
C_l_0: 0.0
C_n_0: 0.0
C_Y_beta: -0.98
C_l_beta: -0.12
C_n_beta: 0.25
C_Y_p: 0.0
C_l_p: -0.26
C_n_p: 0.022
C_Y_r: 0.0
C_l_r: 0.14
C_n_r: -0.35
C_Y_delta_a: 0.0
C_l_delta_a: 0.08
C_n_delta_a: 0.06
C_Y_delta_r: -0.17
C_l_delta_r: 0.105
